app/factories/ClientFactory.js

```diff
@@ -0,0 +1,22 @@
+function ClientFactory(){
+
+  return {
+    createClient: createClient
+  };
+  function createClient(clientInfo){
+    var client;
+
+    client = {
+        id: clientInfo.playerId,
+        username: clientInfo.username,
+        idIcon: clientInfo.idIcon
+    };
+    module.exports = {
+        id: clientInfo.playerId,
+        username: clientInfo.username,
+        idIcon: clientInfo.idIcon
+    };
+    return client;
+  }
+}
+module.exports = ClientFactory();
```

Comment:
What is this entire file even. Any reason why this can't be 

    module.exports = clientInfo => ({
        id: clientInfo.id,
        username: clientInfo.username,
        idIcon: clientInfo.idIcon
    });

Also, even then this seems redundant since it literally makes an object with the exact same structure as the param.